drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw_B/host_rpu_sys_if.h

```diff
@@ -809,6 +826,7 @@ struct rpu_fw_stats {
 #ifndef CONFIG_NRF700X_RADIO_TEST
 	struct rpu_lmac_stats lmac;
 	struct rpu_umac_stats umac;
+	struct nrf_wifi_stats umac_wifi;
```

Comment:
need a better name for this :)

Reply:
Instead of umac_wifi, we can call it interface_stats?

Reply:
umac_data_stats might be appropriate?

Reply:
I guess even the data structure name needs to be more appropriate. Are these statistics consolidated stats for the entire rpu (i.e. umac+lmac+phy)? If so better to call them wifi_stats (I am leaning against data since it also has beacon stats). If this is specific to umac then why can't this be part of struct rpu_umac_stats?

Reply:
@Srinivas1201, Could you please check this?